architecture/core_concepts/routes.adoc

```diff
@@ -163,11 +163,26 @@ plug-in and finally into an HAProxy configuration:
 .HAProxy Router Data Flow
 image::router_model.png[HAProxy Router Data Flow]
 
-*Sticky Sessions*
+[[routes-sticky-sessions]]
+=== Sticky Sessions
 
-Implementing sticky sessions is up to the underlying router configuration. The
-default HAProxy template implements sticky sessions using the `balance source`
-directive which balances based on the source IP. In addition, the template
+Sticky sessions attempts to ensure that user traffic is directed to the same pod in each session, and improves application user experience by cacheing data. For example, for a cluster with five back-end pods, and two load-balanced routers, you can ensure that the same pod receives the web traffic from the same web browser regardless of the router that handles it.
```

Comment:
@mburke5678 +1 on caching.

But I think just "sticky sessions" is ok. It's like a noun group, and it's used that way in other docs.